app/src/main/java/it/quip/android/activity/CreateCircleActivity.java

```diff
@@ -38,6 +38,12 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    public void onBackPressed() {
+        finish();
+        overridePendingTransition(R.anim.zoom_in, R.anim.slide_down);
```

Comment:
When we create the circle, we should also make a transition back to the last activity. Like the other activities that now have `animateOut`, we should do it for this one too